src/main/kotlin/org/misarch/address/graphql/model/UserAddress.kt

```diff
@@ -0,0 +1,34 @@
+package org.misarch.address.graphql.model
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.federation.directives.FieldSet
+import com.expediagroup.graphql.generator.federation.directives.KeyDirective
+import graphql.schema.DataFetchingEnvironment
+import org.misarch.address.graphql.dataloader.UserDataLoader
+import java.util.*
+import java.util.concurrent.CompletableFuture
+
+@GraphQLDescription("A address associated with a user.")
+@KeyDirective(fields = FieldSet("id"))
+class UserAddress(
+    id: UUID,
+    street1: String,
+    street2: String,
+    city: String,
+    postalCode: String,
+    country: String,
+    companyName: String,
+    internal val userId: UUID,
+    @property:GraphQLDescription("If true, this address is archived and can no longer be used.")
+    val isArchived: Boolean
```

Comment:
I think storing this as timestamp would make sense as well to not lose any information…

Reply:
what information do we loose?
but I can add a timestamp if you want

Reply:
The information when it was archived

Reply:
We can use that for example to sort by archived date

Reply:
tbh I still do not really see the point
but if it makes you happy, here you go
added with b366d33212da84946c548f3bf0107c3119988878